src/pandas_profiling/config.py

```diff
@@ -303,6 +303,46 @@ def update(self, updates: dict) -> "Settings":
         return self.parse_obj(self.copy(update=update))
 
 
+class PandasSettings(Settings):
+    pass
+
```

Comment:
this was done to ensure consistency. Settings is the "default settings", and engine-specific settings override certain parts of those default settings. All usages of Settings should be switched to PandasSettings (Not done in this PR)